config/locales/nl.yml

```diff
@@ -290,10 +310,84 @@ nl:
         default: Gebruik als standaard uitgaand e-mailadres
         verification_token: Geverifieerd
 
+      email_template:
+        name: Sjabloon naam
+        from: Van
+        kind: Soort
+        message: Bericht
+        kinds:
+          user_welcome:
+            template_name: Standaard gebruikers welkom
+            message: |
+              <p>
+              Beste [name],
+              <br />
+              </p>
+              <p>
+              Er is voor u een account aangemaakt bij voor BRIMIR, het open source ticket systeem!
```

Comment:
Let's make it noob friendly: leave "open source" out and change BRIMIR to Brimir.